install_gh.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#################################
+# Installs github cli tool
+# Arguments:
+# None
+# Author: github.com/sosaeio
+# Date: Oct 28, 2023
+#################################
+
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+CM=$(perl -e 'print "\xE2\x9C\x94"') #checkmark
+CURRDIR=$PWD
+
+# OS Specific Information
+OSTYPE=$(uname -s)
+GOFILE="go1.21.3.darwin-arm64.pkg"
+
+# Check if the gh tool has been installed
+check_gh=$({ which gh | grep -o gh > /dev/null && echo 1 || echo 0; })
```

Comment:
Why are you doing this here? Do it in the if condtional like:

if (gh installed)
then
echo already installed
else
install gh
fi

Reply:
Did this to just store a 1 or 0 value and use that to check if gh exists in the system